rest/rest-api/build.gradle

```diff
@@ -3,4 +3,6 @@ dependencies {
     compile 'org.jboss.resteasy:resteasy-multipart-provider:3.0.26.Final'
     compile "org.objectweb.proactive:programming-util:${programmingVersion}"
     compile project(':scheduler:scheduler-api')
+//    compile "org.objectweb.proactive:programming-core:${programmingVersion}"
```

Comment:
could you remove the commented out line?

Reply:
sure